src/__tests__/errorHandler.test.ts

```
@@ -0,0 +1,73 @@
+    method: 'GET',
+    url: '/test-db-unavailable',
+  });
+
+  expect(response.statusCode).toBe(503);
+  expect(response.json()).toEqual({
+    success: false,
+    error: 'Database unavailable',
+  });
+});
+
+import { test, expect, beforeAll, afterAll } from 'vitest';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'test'. Consider to delete it or rename it.

'test' is defined here:


(lint/suspicious/noRedeclare)